.github/workflows/ci.yml

```
@@ -63,25 +65,25 @@ jobs:
     steps:
 
     - name: Checkout Code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+
+    - name: Create Build Environment
+      run: cmake -E make_directory ${{runner.workspace}}/build
 
     - name: Configure
-      working-directory: test
-      run: cmake -S . -B build ${{ matrix.cmake_opts }}
+      working-directory: ${{runner.workspace}}/build
+      shell: bash
+      run: cmake -DARGPARSE_BUILD_SAMPLES=ON -DCMAKE_BUILD_TYPE=Debug ${{ matrix.cmake_opts }} $GITHUB_WORKSPACE
```

I think it's better to leave it like the previous implementation, no need to add another step for creating a directory:
```suggestion
      run: cmake . -B build -DARGPARSE_BUILD_SAMPLES=ON -DCMAKE_BUILD_TYPE=Debug ${{ matrix.cmake_opts }}
```

Also setting `shell: bash` is not needed here.